governance/CHARTER.md

```diff
@@ -21,19 +21,13 @@ b.  The scope of the Project includes collaborative development under the Projec
 
 a.  The Technical Steering Committee (the "TSC") will be responsible for all technical oversight of the open source Project.
 
-b.  The TSC voting members are initially the Project's Committers. At the inception of the project, the Committers of the Project will be as set forth within the "CONTRIBUTING" file within the Project's code repository. The TSC may choose an alternative approach for determining the voting members of the TSC, and any such alternative approach will be documented in the CONTRIBUTING file. Any meetings of the Technical Steering Committee will be open to the public, and can be conducted electronically, via teleconference, or in person.
+b.  The TSC voting members are as set forth within the [`README.md` file](README.md) within this repository. The TSC may choose an alternative approach for determining the voting members of the TSC, and any such alternative approach will be documented in the [`README.md` file](README.md). Any meetings of the Technical Steering Committee will be open to the public, and can be conducted electronically, via teleconference, or in person.
 
-c.  TSC projects generally will involve Contributors and Committers. The TSC may adopt or modify roles so long as the roles are documented in the CONTRIBUTING file. Unless otherwise documented:
+c.  TSC projects generally will involve Collaborators, Contributors and Maintainers. The TSC may adopt or modify roles so long as the roles are documented in the [`README.md` file](README.md).
```

Comment:
This allows the TSC to add new roles, without needing to amend the charter, right?

Reply:
I think that was allowed before already, but I'm fixing the name of the file as that's not in the CONTRIBUTING file, actually that file has been moved to the book.

So TSC could already do that in the CONTRIBUTING file, and if we didn't do it then the other rules would apply. I'm removing the other rules as it doesn't look like something needed, as we'll have now the process documented somewhere else. I was doing that to simplify the charter, but I'm fine keeping those lines if we think it's better.

Reply:
Just to clarify, I'm not opposed to this - on the contrary, I would have suggested that we add a line that allows us to define roles in the Readme through regular TSC meetings, since we still need to define some existing roles like Organization admin or infrastructure maintainers.